packages/web3-state/src/Others.ts

```diff
@@ -102,4 +106,17 @@ export class OthersState<ChainId, SchemaType, ProviderType, NetworkType, Transac
     formatSchemaType(schema: SchemaType): string {
         throw new Error('Method not implemented.')
     }
+    createNativeToken(chainId: ChainId): FungibleToken<ChainId, SchemaType> {
+        throw new Error('Method not implemented.')
+    }
+    createERC20Token(
+        chainId: ChainId,
+        address: string,
+        name?: string,
+        symbol?: string,
+        decimals?: number,
+        logoURI?: string,
+    ): FungibleToken<ChainId, SchemaType> {
```

Comment:
```suggestion
    createFungibleToken(
        chainId: ChainId,
        address: string,
        schemaType: SchemaType,
        name?: string,
        symbol?: string,
        decimals?: number,
        logoURI?: string,
    ): FungibleToken<ChainId, SchemaType> {
```

Alongside this method as well add `createNonFungibleToken`.